26-05-2020 16:30:33.003 ERROR [main]  c.a.v.n.c.handler.NPCProcessHandler.saveActivatedNumbers - [Sync File is not found]: Participant "{0}" not found  please contact your Database administrator or check the resource file
com.asset.vodafone.npc.core.exception.NPCException: [Sync File is not found]: Participant "{0}" not found  please contact your Database administrator or check the resource file
	at com.asset.vodafone.npc.core.handler.NPCProcessHandler.saveActivatedNumbers(NPCProcessHandler.java:510)
	at com.asset.vodafone.npc.core.handler.NPCProcessHandler.syncMode(NPCProcessHandler.java:601)
	at com.asset.vodafone.npc.core.handler.NPCProcessHandler.main(NPCProcessHandler.java:655)
	at com.asset.vodafone.npc.core.handler.Test.TestNPCProcessHandler.testMain(TestNPCProcessHandler.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:542)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:770)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:464)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
Caused by: java.io.FileNotFoundException: <?xml version='1.0' encoding='UTF-8'?> <NPCBulkSyncData> <MessageCode>NP Bulk Answer<\MessageCode> <IDNumber>S201802071544470253<\IDNumber> <NumberOfMessages>5<\NumberOfMessages> <TimeStampOfLastBroadCast>20070207121747<\TimeStampOfLastBroadCast> <Comments1>Full Bulk Sync<\Comments1> <Comments2>test done<\Comments2> <ActivatedNumbers> <ActivatedNumber> <IDNumber>P220710021212130208<\IDNumber> <NumberFrom>010010010062<\NumberFrom> <NumberTo>010010010062<\NumberTo> <DonorId>Voda<\DonorId> <RecipientId>Mobi<\RecipientId> <NewRoute>41<\NewRoute> <Action>Port<\Action> <\ActivatedNumber> <ActivatedNumber> <IDNumber>P200712121243380222<\IDNumber> <NumberFrom>010010010124<\NumberFrom> <NumberTo>010010010124<\NumberTo> <DonorId>Voda<\DonorId> <RecipientId>Mobi<\RecipientId> <NewRoute>41<\NewRoute> <Action>Port<\Action> <\ActivatedNumber> <ActivatedNumber> <IDNumber>P200712121242160220<\IDNumber><NumberFrom>010010010122<\NumberFrom> <NumberTo>010010010122<\NumberTo> <DonorId>Voda<\DonorId> <RecipientId>Mobi<\RecipientId> <NewRoute>41<\NewRoute> <Action>Port<\Action> <\ActivatedNumber> <ActivatedNumber> <IDNumber>P200710021211340207<\IDNumber> <NumberFrom>010010010071<\NumberFrom> <NumberTo>010010010071<\NumberTo> <DonorId>Voda<\DonorId> <RecipientId>Mobi<\RecipientId> <NewRoute>41<\NewRoute> <Action>Port<\Action> <\ActivatedNumber> <ActivatedNumber> <IDNumber>P200710021213410210<\IDNumber> <NumberFrom>010010010074<\NumberFrom> <NumberTo>010010010074<\NumberTo> <DonorId>Voda<\DonorId> <RecipientId>Mobi<\RecipientId> <NewRoute>41<\NewRoute> <Action>Port<\Action> <\ActivatedNumber> <ActivatedNumber> <IDNumber>P200712121243000221<\IDNumber> <NumberFrom>010010010123<\NumberFrom> <NumberTo>010010010123<\NumberTo> <DonorId>Voda<\DonorId> <RecipientId>Mobi<\RecipientId> <NewRoute>41<\NewRoute> <Action>Port<\Action> <\ActivatedNumber> <ActivatedNumber> <IDNumber>P200707011647040101<\IDNumber> <NumberFrom>01001001020<\NumberFrom> <NumberTo>01001001020<\NumberTo> <DonorId>Voda<\DonorId> <RecipientId>Mobi<\RecipientId> <NewRoute>41<\NewRoute> <Action>Port<\Action> <\ActivatedNumber> <ActivatedNumber> <IDNumber>P200701200002200156<\IDNumber> <NumberFrom>010010010030<\NumberFrom> <NumberTo>010010010030<\NumberTo> <DonorId>Voda<\DonorId> <RecipientId>Mobi<\RecipientId> <NewRoute>41<\NewRoute> <Action>Port<\Action> <\ActivatedNumber> <ActivatedNumber> <IDNumber>P200801161139470002<\IDNumber> <NumberFrom>01001001006<\NumberFrom><NumberTo>01001001006<\NumberTo> <DonorId>Voda<\DonorId> <RecipientId>Mobi<\RecipientId> <NewRoute>41<\NewRoute> <Action>Port<\Action> <\ActivatedNumber> <\ActivatedNumbers> <\NPCBulkSyncData> (The filename, directory name, or volume label syntax is incorrect)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileInputStream.<init>(FileInputStream.java:93)
	at com.asset.vodafone.npc.core.handler.NPCProcessHandler.saveActivatedNumbers(NPCProcessHandler.java:498)
	... 28 common frames omitted
26-05-2020 16:32:47.652 ERROR [main]  c.a.v.npc.core.service.NPCService.initiateDBConnection - Cannot Establish Database Connection
26-05-2020 16:32:47.663 ERROR [main]  c.a.v.n.c.handler.NPCProcessHandler.saveActivatedNumbers - [NPC_DATABASE_ERROR_01]: Cannot Establish Database Connection
com.asset.vodafone.npc.core.exception.NPCException: [NPC_DATABASE_ERROR_01]: Cannot Establish Database Connection
	at com.asset.vodafone.npc.core.service.NPCService.initiateDBConnection(NPCService.java:86)
	at com.asset.vodafone.npc.core.handler.NPCProcessHandler.saveActivatedNumbers(NPCProcessHandler.java:493)
	at com.asset.vodafone.npc.core.handler.NPCProcessHandler.syncMode(NPCProcessHandler.java:601)
	at com.asset.vodafone.npc.core.handler.NPCProcessHandler.main(NPCProcessHandler.java:655)
	at com.asset.vodafone.npc.core.handler.Test.TestNPCProcessHandler.testMain(TestNPCProcessHandler.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:542)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:770)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:464)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
Caused by: java.sql.SQLRecoverableException: IO Error: Connection reset
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:682)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:711)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:558)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.asset.vodafone.npc.core.service.NPCService.getDBConnection(NPCService.java:114)
	at com.asset.vodafone.npc.core.service.NPCService.initiateDBConnection(NPCService.java:78)
	... 29 common frames omitted
Caused by: java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(SocketInputStream.java:209)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at oracle.net.ns.Packet.receive(Packet.java:311)
	at oracle.net.ns.NSProtocolStream.negotiateConnection(NSProtocolStream.java:153)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:263)
	at oracle.jdbc.driver.T4CConnection.connect(T4CConnection.java:1360)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:486)
	... 37 common frames omitted
26-05-2020 18:41:01.078 ERROR [main]  c.a.v.npc.core.service.NPCService.initiateDBConnection - Cannot Establish Database Connection
26-05-2020 18:41:01.084 ERROR [main]  c.a.v.n.c.handler.NPCProcessHandler.saveActivatedNumbers - [NPC_DATABASE_ERROR_01]: Cannot Establish Database Connection
com.asset.vodafone.npc.core.exception.NPCException: [NPC_DATABASE_ERROR_01]: Cannot Establish Database Connection
	at com.asset.vodafone.npc.core.service.NPCService.initiateDBConnection(NPCService.java:86)
	at com.asset.vodafone.npc.core.handler.NPCProcessHandler.saveActivatedNumbers(NPCProcessHandler.java:493)
	at com.asset.vodafone.npc.core.handler.NPCProcessHandler.syncMode(NPCProcessHandler.java:601)
	at com.asset.vodafone.npc.core.handler.NPCProcessHandler.main(NPCProcessHandler.java:655)
	at com.asset.vodafone.npc.core.handler.Test.TestNPCProcessHandler.testMain(TestNPCProcessHandler.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:542)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:770)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:464)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
Caused by: java.sql.SQLRecoverableException: IO Error: Socket read timed out
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:673)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:711)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:558)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.asset.vodafone.npc.core.service.NPCService.getDBConnection(NPCService.java:114)
	at com.asset.vodafone.npc.core.service.NPCService.initiateDBConnection(NPCService.java:78)
	... 27 common frames omitted
Caused by: oracle.net.ns.NetException: Socket read timed out
	at oracle.net.ns.Packet.receive(Packet.java:350)
	at oracle.net.ns.NSProtocolStream.negotiateConnection(NSProtocolStream.java:153)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:263)
	at oracle.jdbc.driver.T4CConnection.connect(T4CConnection.java:1360)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:486)
	... 35 common frames omitted
26-05-2020 19:54:00.051 ERROR [main]  c.a.v.npc.core.service.NPCService.initiateDBConnection - Cannot Establish Database Connection
26-05-2020 19:54:00.060 ERROR [main]  c.a.v.n.c.handler.NPCProcessHandler.saveActivatedNumbers - [NPC_DATABASE_ERROR_01]: Cannot Establish Database Connection
com.asset.vodafone.npc.core.exception.NPCException: [NPC_DATABASE_ERROR_01]: Cannot Establish Database Connection
	at com.asset.vodafone.npc.core.service.NPCService.initiateDBConnection(NPCService.java:86)
	at com.asset.vodafone.npc.core.handler.NPCProcessHandler.saveActivatedNumbers(NPCProcessHandler.java:493)
	at com.asset.vodafone.npc.core.handler.NPCProcessHandler.syncMode(NPCProcessHandler.java:601)
	at com.asset.vodafone.npc.core.handler.NPCProcessHandler.main(NPCProcessHandler.java:655)
	at com.asset.vodafone.npc.core.handler.Test.TestNPCProcessHandler.testMain(TestNPCProcessHandler.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:542)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:770)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:464)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
Caused by: java.sql.SQLRecoverableException: IO Error: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:673)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:711)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:558)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.asset.vodafone.npc.core.service.NPCService.getDBConnection(NPCService.java:114)
	at com.asset.vodafone.npc.core.service.NPCService.initiateDBConnection(NPCService.java:78)
	... 27 common frames omitted
Caused by: oracle.net.ns.NetException: The Network Adapter could not establish the connection
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:445)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:464)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:594)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:229)
	at oracle.jdbc.driver.T4CConnection.connect(T4CConnection.java:1360)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:486)
	... 35 common frames omitted
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:162)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:133)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:411)
	... 40 common frames omitted
